scrapers/FemDomEmpire.yml

```diff
@@ -0,0 +1,31 @@
+name: "FemDomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="title"]/h3
+      Date:
+        selector: //div[@class="videoInfo clear"]/p

```

Comment:
the selector matches a lot more paragraphs than the date one
```yaml
       selector: //p[span[contains(.,"Date Added")]]
```
only returns the date paragraph